rotkehlchen/exchanges/kucoin.py

```diff
@@ -116,10 +117,9 @@ def __str__(self) -> str:
 PAGINATED_CASES = (KucoinCase.OLD_TRADES, KucoinCase.TRADES, KucoinCase.DEPOSITS, KucoinCase.WITHDRAWALS)  # noqa: E501
 
 
-def _serialize_ts(case: KucoinCase, time: int) -> int:
-    if case == KucoinCase.OLD_TRADES:
-        return time
-    return time * 1000
+def _serialize_ts(time: Timestamp) -> Timestamp:
```

Comment:
If it's just a multiplication by 1000 do we really need this method? We don't use such methods in other exchanges where similar happens.

Reply:
had it to avoid doing manually the conversion from seconds to miliseconds in case they change in the future but I can just do the multiplication directly where it is used